cnd/src/btsieve/bitcoin/bitcoind_connector.rs

```diff
@@ -46,15 +46,9 @@ impl LatestBlock for BitcoindConnector {
             .client
             .get(self.chaininfo_url.clone())
             .send()
-            .map_err(|e| {
-                log::error!("Error when sending request to bitcoind");
-                Self::Error::Reqwest(e)
-            })
+            .map_err(Self::Error::Reqwest)
             .and_then(move |mut response| {
-                response.json::<ChainInfo>().map_err(|e| {
-                    log::error!("Error when deserialising the response from bitcoind");
```

Comment:
Why removing this error?

Reply:
Because the error is propagated back up the call chain.  Following the adage 'either return the error or log it'.  Happy to be corrected, ether on the truth of this adage or on this particular case.